base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleDataResponse.java

```diff
@@ -1,597 +1,649 @@
+    public GetVehicleDataResponse(@NonNull Boolean success, @NonNull Result resultCode) {
+        this();
+        setSuccess(success);
+        setResultCode(resultCode);
+    }
+
+    public GetVehicleDataResponse setGps(GPSData gps) {
+        setParameters(KEY_GPS, gps);
+        return this;
+    }
+
+    public GPSData getGps() {
+        return (GPSData) getObject(GPSData.class, KEY_GPS);
+    }
+
+    public GetVehicleDataResponse setSpeed(Double speed) {
+        setParameters(KEY_SPEED, speed);
+        return this;
+    }
+
+    public Double getSpeed() {
+        Object object = getParameters(KEY_SPEED);
+        return SdlDataTypeConverter.objectToDouble(object);
+    }
+
+    public GetVehicleDataResponse setRpm(Integer rpm) {
+        setParameters(KEY_RPM, rpm);
+        return this;
+    }
+
+    public Integer getRpm() {
+        return getInteger(KEY_RPM);
+    }
+
+    /**
+     * Sets Fuel Level State
+     *
+     * @param fuelLevelState a ComponentVolumeStatus related to FuelLevel State
+     */
+    @Deprecated
+    public GetVehicleDataResponse setFuelLevelState(ComponentVolumeStatus fuelLevelState) {
+        setParameters(KEY_FUEL_LEVEL_STATE, fuelLevelState);
+        return this;
+    }
+
+    /**
+     * Gets Fuel Level State
+     *
+     * @return a ComponentVolumeStatus related to FuelLevel State
+     */
+    @Deprecated
+    public ComponentVolumeStatus getFuelLevelState() {
+        return (ComponentVolumeStatus) getObject(ComponentVolumeStatus.class, KEY_FUEL_LEVEL_STATE);
+    }
+
+    /**
+     * Sets the fuelLevel.
+     *
+     * @param fuelLevel The fuel level in the tank (percentage). This parameter is deprecated starting RPC Spec
+     *                  7.0, please see fuelRange.
+     * @since SmartDeviceLink 7.0.0
+     */
+    @Deprecated
+    public GetVehicleDataResponse setFuelLevel(Double fuelLevel) {
+        setParameters(KEY_FUEL_LEVEL, fuelLevel);
+        return this;
+    }
+
+    /**
+     * Gets the fuelLevel.
+     *
+     * @return Double The fuel level in the tank (percentage). This parameter is deprecated starting RPC Spec
+     * 7.0, please see fuelRange.
+     * @since SmartDeviceLink 7.0.0
+     */
+    @Deprecated
+    public Double getFuelLevel() {
+        Object object = getParameters(KEY_FUEL_LEVEL);
+        return SdlDataTypeConverter.objectToDouble(object);
+    }
+
+    public GetVehicleDataResponse setInstantFuelConsumption(Double instantFuelConsumption) {
+        setParameters(KEY_INSTANT_FUEL_CONSUMPTION, instantFuelConsumption);
+        return this;
+    }
+
+    public Double getInstantFuelConsumption() {
+        Object object = getParameters(KEY_INSTANT_FUEL_CONSUMPTION);
+        return SdlDataTypeConverter.objectToDouble(object);
+    }
+
+    /**
+     * Sets the externalTemperature.
+     *
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
```

Comment:
@kostyaBoss When using the RPC generator tool, on line 222 I got 7.1.0. I would replace 7.1 with 7.1.0.